carts/models.py

```diff
@@ -0,0 +1,65 @@
+"""
+Models for cart app
+"""
+from decimal import Decimal
+
+from django.core.validators import MinValueValidator
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+from products.models import AuditTimeStamp, Product
+from users.models import User
+
+
+class Cart(AuditTimeStamp):
+    """
+    Cart model
+    """
+
+    objects = models.Manager()
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name="cart")
+    total_bill = models.DecimalField(
+        _("Total Payable"),
+        max_digits=20,
+        validators=(MinValueValidator(1),),
+        decimal_places=2,
+        default=Decimal(0),
+    )
+
+    def update_bill(self):
+        """
+        Updates the bill by traversing all the cart items
+        Returns:
+            None
+        """
+        # pylint: disable=no-member
+        total = Decimal(0)
+        for item in self.cart_items.all():
+            total += item.product.price * Decimal(item.quantity)
+        self.total_bill = Decimal(total)
+        self.save()
+
+    def __str__(self):
+        return f"{self.user.name}'s Cart"
+
+
+class CartItem(AuditTimeStamp):
+    """
+    Cart item model
+    """
+
+    objects = models.Manager()
```

Comment:
Do we need to define this?

Reply:
We do not need to do this but if we do not do this then pylint gives error `no-member` wherever we use `Model.objects`. Should i instead use `# pylint: disable=no-member` ?

Reply:
Sure, you can use the pylint ignore.